components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -58,6 +60,40 @@ sealed class SystemAction : BrowserAction() {
     ) : SystemAction()
 }
 
+/**
+ * [BrowserAction] implementations related to updating the list of [ClosedTabSessionState] inside [BrowserState].
+ */
+sealed class RecentlyClosedAction : BrowserAction() {
+    /**
+     * Initializes the [BrowserState.closedTabs] state.
+     */
+    object InitializeRecentlyClosedState : RecentlyClosedAction()
```

Comment:
I am taking this from the container feature - is there already a place that we initialize these types of things in AC and/or Fenix?

Reply:
Not yet afaik. The plan was to, for now, dispatch the action in the app, for example right after creating the store, e.g. something like

```Kotlin
val store by lazy {
  BrowserStore(
    // ...
  ).apply {
    dispatch(...)
  }
}
```

We were debating about introducing a generic "init" action that the store could dispatch itself when ready - so that a middleware can listen to that and no custom dispatch and custom action would be needed. But we haven't build anything like that yet.